application-tour-ui/src/main/resources/TourCode/TourJS.xml

```diff
@@ -276,8 +274,6 @@ require(['jquery', 'xwiki-meta'], function ($, xm) {
       tour.init();
       if (forceStart) {
         tour.restart();
-      } else {
-        tour.start();
```

Comment:
I was trying to solve this issue, and based myself off this:
http://stackoverflow.com/questions/27106349/bootstrap-tour-doesnt-remember-where-i-left#27117439